ios/uploadDebugSymbols.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-export SENTRY_PROPERTIES=sentry.properties
```

Comment:
I don't see anything in their changelog about no longer needing this, but I trust that you've tested this